vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -43,7 +86,7 @@ func CreateProcessInComputeSystem(id string, params CreateProcessParams) (proces
 	if err != nil {
 		err = fmt.Errorf(title+" - Failed conversion of id %s to pointer %s", id, err)
 		logrus.Error(err)
-		return 0, err
+		return
```

Comment:
the `err` shadowing is hard to reason about. I never know what Go does with this and it's not obvious. I would prefer to return expicitly `return nil, nil, nil, err`